src/test/java/com/teamsfinder/userwriteservice/user/controller/UserControllerTest.java

```diff
@@ -0,0 +1,72 @@
+package com.teamsfinder.userwriteservice.user.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.teamsfinder.userwriteservice.user.dto.EditUserDto;
+import org.junit.Before;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.testcontainers.shaded.com.fasterxml.jackson.core.JsonProcessingException;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.SerializationFeature;
+
+import java.net.URI;
+import java.util.ArrayList;
+
+@AutoConfigureMockMvc
+@Sql(scripts = "classpath:import.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+@Sql(scripts = "classpath:clear.sql", executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+class UserControllerTest extends TestContainer {
```

Comment:
jak juz to zrób base class typu IntegrationTest który będzie zajmował się czyszczeniem bazy po testach, przykład taki masz w moim repo jdk17-template zdaje mi się

Reply:
Nie moge tego poprostu do testcontainer przeniesc?